### ISSUE TITLE
java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.String

### ISSUE THREAD

**Issue by WouterFlorijn** [AppsFlyerSDK/appsflyer-react-native-plugin#123]
# Report

## Plugin Version

1.4.4

## On what Platform are you having the issue?

android

## What did you do?

Call `appsFlyer.setCustomerUserId(userId)` with a number as `userId`.

## What did you expect to happen?

Either the number is cast to string, or I get a clear JS error stating that the id should be a string.

## What happened instead?

I get a cryptic Java exception:

```
11-28 16:26:38.068 21070 21131 E AndroidRuntime: FATAL EXCEPTION: mqt_native_modules
11-28 16:26:38.068 21070 21131 E AndroidRuntime: Process: com.my.app, PID: 21070
11-28 16:26:38.068 21070 21131 E AndroidRuntime: java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.String
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.ReadableNativeArray.getString(ReadableNativeArray.java:135)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.JavaMethodWrapper$5.extractArgument(JavaMethodWrapper.java:73)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.JavaMethodWrapper$5.extractArgument(JavaMethodWrapper.java:69)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:359)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:158)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:873)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:99)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:193)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232)
11-28 16:26:38.068 21070 21131 E AndroidRuntime:        at java.lang.Thread.run(Thread.java:764)
11-28 16:48:37.889 24918 24982 E AndroidRuntime: FATAL EXCEPTION: mqt_native_modules
11-28 16:48:37.889 24918 24982 E AndroidRuntime: Process: com.my.app, PID: 24918
11-28 16:48:37.889 24918 24982 E AndroidRuntime: java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.String
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.ReadableNativeArray.getString(ReadableNativeArray.java:135)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.JavaMethodWrapper$5.extractArgument(JavaMethodWrapper.java:73)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.JavaMethodWrapper$5.extractArgument(JavaMethodWrapper.java:69)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:359)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:158)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:873)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:99)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:193)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232)
11-28 16:48:37.889 24918 24982 E AndroidRuntime:        at java.lang.Thread.run(Thread.java:764)
```

## Please provide any other relevant information.

Easy to fix using `toString()` of course, but took way too long to debug due to the lack of exception handling.

**Comment by n3xrodrigo** [comment]
Same with `updateServerUninstallToken`

**Comment by amit-kremer93** [comment]
fixed on 6.0.31
